### ISSUE TITLE
Add yearly population to summary response from /incidents/count

### ISSUE THREAD

**Issue by jeremiak** [18F/crime-data-api#290]
So that we can accurately compute the yearly rate for a crime in a place, we need to know the population. It would be helpful if that population count is returned in the same response of `incidents/count`

**Comment by harrisj** [comment]
So, if I understand it correctly, compute population like this:

|Group by|Calculation|
|-----------|-----------|
|None|Add up `ref_state_population` for latest data year|
|by state|The `population` for the matching `ref_state_population`|
|by county|The `population` for the matching `ref_county_population`|
|by agency|The sum of the `population` from `ref_agency_county`|

**Comment by harrisj** [comment]
I haven't had a chance to start this yet. Sorry, @jeremiak. Will get to this today.

One wrinkle I have discovered is that the `ref_state_population` seems to be lagging the `ref_county_population` and `ref_agency_county` by a few years (see #292 for details). I still think I should use the `ref_state_population` figure rather than summing all the counties mapped to the state, since there might be other reasons why the state population would not be a sum of the counties (I am not sure if tribes are counted at the state level for instance)

**Comment by jeremiak** [comment]
No worries @harrisj!

I agree that we should use `ref_state_population`, that sounds the simplest.

**Comment by LarryBafundo** [comment]
@harrisj could you please provide an update on this prior to sprint planning tomorrow? thanks

**Comment by harrisj** [comment]
So, I have population figured out for states and counties using their preferred approaches in #341. I'm just trying to figure out how to join in the `CdeParticipationRate` table as appropriate in the IncidentsCount query. It's a bit obscure how this code works...

**Comment by harrisj** [comment]
So, I'm not confident in how the full DB will work with my original plan of doing a join across from the `RetaMonthOffenseSubcatSummary` and `ParticipationRate` tables and the full-size DB. I know it's not an ideal answer, but could you just download the data for the state using the `/geo/states/STATE_ID/participation` endpoint in #341. This returns population counts by year in records like so:

``` json
{
"covered_population": 151120,
"data_year": 1960,
"reporting_agencies": 12,
"reporting_rate": 1,
"total_agencies": 12,
"total_population": 153942
}
```

(these numbers are really small because I'm working with a test subset of the DB). I am hoping to get this PR merged to production today and to run the code to generate the participation_rate table today or tomorrow at the latest

**Comment by jeremiak** [comment]
I'm totally fine getting this data from another endpoint. I made a small request for changing `data_year` to `year` in ^^, and other than that I just have a question on the `state_id` value: Is this the same as the [codes we already use for other endpoints](https://github.com/18F/crime-data-explorer/blob/master/src/util/api.js#L34-L86)?

**Comment by harrisj** [comment]
It should be, and I pushed some code to make that change for you. Thanks for your understanding!

**Comment by harrisj** [comment]
This was fixed in #341
